src/Components/Updater/Installer.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * This file is part of Laravel Zero.
+ *
+ * (c) Nuno Maduro <enunomaduro@gmail.com>
+ *
+ *  For the full copyright and license information, please view the LICENSE
+ *  file that was distributed with this source code.
+ */
+
+namespace LaravelZero\Framework\Components\Updater;
+
+use Illuminate\Support\Facades\File;
+use LaravelZero\Framework\Components\AbstractInstaller;
+
+/**
+ * @internal
+ */
+final class Installer extends AbstractInstaller
+{
+    /**
+     * {@inheritdoc}
+     */
+    protected $name = 'install:updater';
```

Comment:
How about `install:build-updater`?

Reply:
Don't you think that this name will be confusing for others?
As they might connect the `build` with `box` and think that this will update the box binary?